testing/kuttl/e2e/delete/13-delete-cluster-and-check.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # Delete all previous events
+  - script: |
+      kubectl delete --namespace="${NAMESPACE}" events --all 
```

Comment:
:question:  I'm curious about what happens when all the tests are run in the same namespace. I don't think we currently do a lot of (any?) event checking with our other tests, but I could see this being an issue in the future. Did you happen to run the tests in a single namespace during your testing?

Reply:
My initial thought here was 
a) most of the time, we run several tests at once, but in different namespaces -- I think actually that some of our other KUTTL tests would fail if we ran several tests at once in the same namespace just because our selectors aren't specific (though I don't think that's true of all the tests)
b) when we do run in the same namespace (which is how I used to test in AWS, since I needed a secret for pulling images), I would make sure to turn it to run only a single test at a time.

That said, I think I can obviate the need for this by getting the events for the specific pod and then comparing the created timestamps...

Reply:
Changed to avoid the need for this, which I think is better testing practice regardless.